src/service-worker/ServiceWorker.ts

```diff
@@ -408,13 +408,14 @@ export class ServiceWorker {
 
     notification.heading = notification.heading ? notification.heading : defaultTitle;
     notification.icon = notification.icon ? notification.icon : (defaultIcon ? defaultIcon : undefined);
-    var extra: any = {};
+    const extra: any = {};
     extra.tag = notification.tag || appId;
-    if (persistNotification === 'force') {
+    if (persistNotification === 'force')
       extra.persistNotification = true;
-    } else {
+    else if (persistNotification == null)
+      extra.persistNotification = true;
+    else
```

Comment:
this whole block can probably be 

```
extra.persistNotification = (persistNotification === false) ? false : true;
```
```

Reply:
Found `extra.persistNotification = persistNotification !== false;`